analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/Roslyn/RestrictDeserializedTypes.cs

```diff
@@ -35,6 +35,28 @@ public override bool ShouldExecute()
         return walker.Result;
     }
 
+    protected override SyntaxNode BindToTypeDeclaration(IOperation operation) =>
```

Comment:
I find this name confusing - because of the verb `Bind` it sounds like it `Binds` something, when my understanding is that it tries to find the declaration of a particular method. Is that correct?